src/docs/product/performance/web-vitals.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Web Vitals"
+sidebar_order: 6
+description: "Learn more about Web Vitals in Sentry and how they give you a better understanding of your application's overall health.
+---
+
+Some transactions will additionally contain web vital information. With this view, you'll be able to examine a summary of these web vitals.
+
+A summary is generated for each of the following web vitals.
+
+- [First Paint (FP)](#first-paint-fp)
+- [First Contentful Paint (FCP)](#first-contentful-paint-fcp)
+- [Largest Contentful Paint (LCP)](#largest-contentful-paint-lcp)
+- [First Input Delay (FID)](#first-input-delay-fid)
+- [Cumulative Layout Shift (CLS)](#cumulative-layout-shift-cls)
+
+## Baseline Value
+
+The baseline value for each web vital is the 75th percentile of the observed data points. It is compared against a failure threshold to give a pass or fail for each web vital.
+
+### Thresholds
+
+| Web Vital                                                 | Threshold        |
+| --------------------------------------------------------- | ---------------- |
+| [First Paint](#first-paint-fp)                            | 3 seconds        |
+| [First Contentful Paint](#first-contentful-paint-fcp)     | 3 seconds        |
+| [Largest Contentful Paint](#largest-contentful-paint-lcp) | 4 seconds        |
+| [First Input Delay](#first-input-delay-fid)               | 300 milliseconds |
+| [Cumulative Layout Shift](#cumulative-layout-shift-cls)   | 0.25             |
+
+## Distribution Histogram
+
+A histogram visualization showing the distribution of the data can help you identify and diagnose frontend performance problems by revealing anomalies.
+
+By default, outliers will be excluded from the histograms to provide a more informative view of these vitals. Outliers are determined by using the [upper outer fence](https://www.itl.nist.gov/div898/handbook/prc/section1/prc16.htm) as the upper bound, and any data points above the upper bound are deemed to be an outlier. If you wish to see all of the data available, you can still do so.
```

Comment:
Not sure about this link as a reference. Can't find another reference right now.

Reply:
https://www.thoughtco.com/what-are-inner-and-outer-fences-3126374 maybe?

Reply:
Using this wikipedia link instead https://en.wikipedia.org/wiki/Outlier#Tukey's_fences